pkgs/by-name/ph/phylophlan/package.nix

```diff
@@ -0,0 +1,65 @@
+{
+  lib,
+  fetchFromGitHub,
+  raxml,
+  mafft,
+  trimal,
+  blast,
+  diamond,
+  python3Packages,
+}:
+let
+  finalAttrs = {
+    pname = "phylophlan";
+    version = "3.1.1";
+    pyproject = true;
+
+    src = fetchFromGitHub {
+      owner = "biobakery";
+      repo = "phylophlan";
+      rev = finalAttrs.version;
```

Comment:
`rev = "refs/tags/${finalAttrs.version}"` would be safer in case there is ever a branch matching the tag name